### PR TITLE
Some bug fixes

### PR DESCRIPTION
**Pull request by NicolasEYSSERIC**

### CODE REVIEW

File: core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java [hunk 1]
```diff
@@ -153,20 +153,25 @@ public void displayScripts(PrintWriter out, FieldTemplate template, PagesContext
...
     String contentLanguage = I18NHelper.checkLanguage(pageContext.getContentLanguage());
 
-    code = getContent(pageContext.getComponentId(), pageContext.getObjectId(), template.
-        getFieldName(), code, contentLanguage);
+    String code = getContent(pageContext.getComponentId(), pageContext.getObjectId(), fieldName,
+        contentLanguage);
+
+    if (code == null) {
+      // get content in other languages
+      List<String> languages = new ArrayList<String>(I18NHelper.getAllSupportedLanguages());
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java]: MINOR Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2293)

File: core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java [hunk 2]
```diff
@@ -510,8 +519,7 @@ public void index(FullIndexEntry indexEntry, String key, String fieldName, TextF
 
   public String duplicateContent(Field field, FieldTemplate template, ForeignPK from, ForeignPK to,
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java]: MAJOR Remove this unused method parameter "field". [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)

**Comment by silveruser** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L355](https://github.com/Silverpeas/Silverpeas-Core/blob/4b44120e45e591c4083e89cb4bb33473ad43dbe7/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L355): Define a constant instead of duplicating this literal "\").selectedIndex;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L357](https://github.com/Silverpeas/Silverpeas-Core/blob/4b44120e45e591c4083e89cb4bb33473ad43dbe7/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L357): Define a constant instead of duplicating this literal "\").options[index].value;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by silveruser** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L355](https://github.com/Silverpeas/Silverpeas-Core/blob/d07a281485989fe270f6358502692f20241d512b/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L355): Define a constant instead of duplicating this literal "\").selectedIndex;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L357](https://github.com/Silverpeas/Silverpeas-Core/blob/d07a281485989fe270f6358502692f20241d512b/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L357): Define a constant instead of duplicating this literal "\").options[index].value;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java [hunk 3]
```diff
@@ -508,10 +517,9 @@ public void index(FullIndexEntry indexEntry, String key, String fieldName, TextF
     }
   }
 
-  public String duplicateContent(Field field, FieldTemplate template, ForeignPK from, ForeignPK to,
+  public String duplicateContent(FieldTemplate template, ForeignPK from, ForeignPK to,
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java]: INFO Remove this use of "ForeignPK"; it is deprecated. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
INFO Remove this use of "ForeignPK"; it is deprecated. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by silveruser** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L355](https://github.com/Silverpeas/Silverpeas-Core/blob/f90f8e8c029a0aa9599d3eff261ba44abd47dc37/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L355): Define a constant instead of duplicating this literal "\").selectedIndex;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [WysiwygFCKFieldDisplayer.java#L357](https://github.com/Silverpeas/Silverpeas-Core/blob/f90f8e8c029a0aa9599d3eff261ba44abd47dc37/core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java#L357): Define a constant instead of duplicating this literal "\").options[index].value;" 3 times. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by NicolasEYSSERIC** [comment]
Close this PR. Another one is coming soon...

**Pull request closed without merging**